etc/preload-daemon

```diff
@@ -23,7 +23,7 @@ fi
 source "${DIRECTORY}/api" || error "failed to source ${DIRECTORY}/api"
 
 #for mktimestamps function
-source "${DIRECTORY}/preload" source
+source "${DIRECTORY}/preload" "source"
```

Comment:
What difference does this change make?

Reply:
it fixes the bug. the bug without the quotes is source is not passed as a string like you expect. it gets passed as a command or something. which causes the preload script to not exit here where you expect:
https://github.com/Botspot/pi-apps/blob/52a1d52acae1b29c6141645d275e21de438fbb8e/preload#L34